hack/test-extended/default/run.sh

```diff
@@ -144,12 +145,13 @@ oc create -n openshift -f examples/image-streams/image-streams-centos7.json --co
 registry="$(dig @${API_HOST} "docker-registry.default.svc.cluster.local." +short A | head -n 1)"
 echo "[INFO] Registry IP - ${registry}"
 
-echo "[INFO] Starting extended tests ..."
-
-# time go test ./test/extended/ #"${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testdir}"  | grep --color=never -E "${1-Test}" | xargs -I {} -n 1 bash -c "exectest {} ${@:2}" # "${testexec}" -test.run="^{}$" "${@:2}"
 echo "[INFO] MASTER IP - ${MASTER_ADDR}"
 echo "[INFO] SERVER CONFIG PATH - ${SERVER_CONFIG_DIR}"
+echo "[INFO] Starting extended tests ..."
 
 # Run the tests
-KUBECONFIG="${ADMIN_KUBECONFIG}" \
-  ginkgo -progress -focus="default:" -p ./test/extended/extended.test
+pushd ${OS_ROOT}/test/extended >/dev/null
+export KUBECONFIG="${ADMIN_KUBECONFIG}"
+export EXTENDED_TEST_PATH="${OS_ROOT}/test/extended"
```

Comment:
@deads2k before you ask, this is another caveat of Ginkgo parallel test execution... it changes PWD for 'nodes' that run the test.. so the first node PWD is correct, but the second is reset to `_output/bin/`....

I fixed this by explicitly telling where is the root folder for extended tests, so the test cases can rely on it when reading the fixture files.